lib/dataElementListener.js

```diff
@@ -535,6 +535,9 @@ class DataElementImporter extends SHRDataElementParserListener {
   }
 
   pushCurrentDefinition() {
+    if (this._specs.dataElements.findByIdentifier(this._currentDef.identifier) != null) {
+      logger.error('Name "%s" already exists. ERROR_CODE:11033', this._currentDef.identifier._name);
```

Comment:
I'm sorry for missing this before, but you're using the internal property `_name` to get the name from the identifier.  `Identifier` actually has a public getter for name ([here](https://github.com/standardhealth/shr-models/blob/master/lib/models.js#L595)).  Since you're accessing it from another class (and another module), the preferred approach is `this._currentDef.identifier.name` rather than `this._currentDef.identifier._name`.

Reply:
A good rule of thumb that works _most_ of the time is that internal properties should only be called on `this` -- so `this._currentDef` is fine, but `foo._currentDef` would be discouraged in most cases.